src/ess/reflectometry/load.py

```diff
@@ -1,7 +1,43 @@
+import h5py
 import scippnexus as snx
 
 
-def load(filepath, *paths: str):
+def _load(filepath, *paths: str):
     with snx.File(filepath, 'r') as f:
         for path in paths:
-            yield f[path][()]
+            if path is not None:
+                yield f[path][()]
+            else:
+                yield None
+
+
+def _convert_nxpaths(filepath, *paths: str):
+    with h5py.File(filepath, 'r') as f:
+        for path in paths:
+            raw_path = []
+            current = f
+            for step in path.split('/'):
+                if step.startswith('NX'):
+                    for key, subgroup in current.items():
+                        if subgroup.attrs.get('NX_class') == step:
+                            current = subgroup
+                            raw_path.append(key)
+                            break
```

Comment:
This is a bit of a hack. Note that there could be, e.g., multiple `Nxentry` groups. Maybe make https://github.com/scipp/essreduce/blob/fcf21ddf98acd8d3d99076119ba35f4c19286334/src/ess/reduce/nexus.py#L270-L289 public and use it here?